features/admin.actions.v1/pages/action-configuration-page.tsx

```diff
@@ -0,0 +1,428 @@
+                                isCreateFormState={ showCreateForm }
+                                mutate={ mutateActions }
+                            />
+                        </Grid.Column>
+                    </Grid.Row>
+                </Grid>
+            }
+            { !isLoading && !showCreateForm && (
+                <Show
+                    when={ featureConfig?.actions?.scopes?.delete }
+                >
+                    <DangerZoneGroup
+                        sectionHeader={ t("console:manage.features.actions.dangerZoneGroup.header") }
+                    >
+                        <DangerZone
+                            data-componentid={ `${ _componentId }-delete-action-of-type-${ actionTypeApiPath}` }
+                            actionTitle={
+                                t("console:manage.features.actions.dangerZoneGroup.revertConfig.actionTitle")
+                            }
+                            header={ t("console:manage.features.actions.dangerZoneGroup.revertConfig.heading") }
+                            subheader={
+                                t("console:manage.features.actions.dangerZoneGroup.revertConfig.subHeading")
+                            }
+                            onActionClick={ (): void => {
+                                setOpenRevertConfigModal(true);
+                            } }
+                        />
+                    </DangerZoneGroup>
+                    <ConfirmationModal
+                        primaryActionLoading={ isActionsLoading || !actions || !Array.isArray(actions) }
+                        data-componentid={ `${ _componentId }-revert-confirmation-modal` }
+                        onClose={ (): void => setOpenRevertConfigModal(false) }
+                        type="negative"
+                        open={ isOpenRevertConfigModal }
+                        assertionHint={ t("console:manage.features.actions.confirmationModal.assertionHint") }
+                        assertionType="checkbox"
+                        primaryAction={ t("common:confirm") }
+                        secondaryAction={ t("common:cancel") }
+                        onSecondaryActionClick={ (): void => setOpenRevertConfigModal(false) }
+                        onPrimaryActionClick={ (): void => {
+                            deleteAction(actionTypeApiPath, actionInitialValues.id)
+                                .then(() => {
+                                    handleSuccess(ActionsConstants.DELETE);
+                                    history.push(AppConstants.getPaths().get("ACTIONS"));
+                                })
+                                .catch((error: AxiosError) => {
+                                    handleError(error, ActionsConstants.DELETE);
+                                })
+                                .finally(() => {
+                                    setOpenRevertConfigModal(false);
+                                });
```

Comment:
Move this to a separate function.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/5e3564e0797c09f4f20b71837b928bc9fa57c847